utils/src/MurmurHash3.cpp

```diff
@@ -0,0 +1,442 @@
+//-----------------------------------------------------------------------------
+// MurmurHash3 was written by Austin Appleby, and is placed in the public
+// domain. The author hereby disclaims copyright to this source code.
+
+// Note - The x86 and x64 versions do _not_ produce the same results, as the
```

Comment:
Does the arm64 version a) exist and b) produce the same result as for x64?

Reply:
Good question, I don't know. Maybe @hegner can say something on that?

Reply:
Probably doesn't matter, because only `MurmurHash3_x86_32` is used. Which means less than optimal performance, but probably doesn't matter either.